### ISSUE TITLE
Agenda: Community Council (CC), 23 November 2021

### ISSUE THREAD

**Issue by stakervali** [grincc/agenda#25]
Solicit suggestions for agenda items for the first Governence meeting to be held on Tuesday November 23 @ 10:00 UTC in grincoin#general channel on [Keybase](https://keybase.io/team/grincoin). Please comment to provide topics or suggestions.
(Meeting can be followed on @keybase-ro channel - GRIN community discord server via GRIN discord bridge) [discord invite](https://discord.gg/5p4vCQY9km)

## Proposed Agenda

- ...
- ...

**Comment by gorfist** [comment]
Considering a bounty for adding Grin to THORchain native coin alongside monero & bitcoin

**Comment by MCM-Mike** [comment]
Transaction logs need to be generated for public to see.

**Comment by MCM-Mike** [comment]
Short update on GRIN-Exchange progress:

- BisQ (updates!)
- KuCoin (updates!)
- Gate . io (updates!)
- Hotbit (updates!)

**Comment by 2ro** [comment]
I have been unsuccessful in asking the community for the remaining 100ツ needed to run slatepacks.com -- Can I please bring this up to the CC and ask for your official input regarding funding the GRIN marketplace I made

**Comment by davidtavarez** [comment]
> I have been unsuccessful in asking the community for the remaining 100ツ needed to run slatepacks.com for another 5 months -- Can I please bring this up to the CC and ask for your official input regarding funding the GRIN marketplace I made?

@2ro DM me in Telegram.

**Comment by Anynomouss** [comment]
Add topic:
- Proposal for the Grin CC to buy around 0.5BTC 1 BTC (1BTC = 50k € ) worth in GRIN. 
_The purpose it to buy ahead and make sure we always have around 0.5BTC worth of GRIN for paying Funding Requets. This first buy in is a bridge fund till the miners arrive and produce enough Grin for funding Requests_

**Comment by boorishpea** [comment]
https://github.com/mimblewimble/grin-pm/issues/385#issuecomment-971358022

**Comment by stakervali** [comment]
> 
> 
> [mimblewimble/grin-pm#385 (comment)](https://github.com/mimblewimble/grin-pm/issues/385#issuecomment-971358022)

Hi @boorishpea , this topic is more likely to be discussed in dev channel but if you still would like it to be discussed in the community meeting, I can add it to agenda.

**Comment by boorishpea** [comment]
> > [mimblewimble/grin-pm#385 (comment)](https://github.com/mimblewimble/grin-pm/issues/385#issuecomment-971358022)
> 
> Hi @boorishpea , this topic is more likely to be discussed in dev channel but if you still would like it to be discussed in the community meeting, I can add it to agenda.

Yes I think you should discuss it, there should be no reason why we are forcing required user input, if it's for security, it should be discussed. It will be quick to implement will make it much easier for scripters.

**Comment by davidtavarez** [comment]
> funding for grinventions python and php and javascript implementations for meta mask esque browser plugin and easy checkout system for any website

we can not offer funding, we can be asked; please read this:

https://github.com/grincc/docs/blob/main/funding_requests.md

**Comment by scilio** [comment]
I would like to request formal acceptance of milestone 1 and council approval to start on milestone 2.

https://forum.grin.mw/t/request-for-funding-scilio-coinswap-implementation/9149
https://github.com/mimblewimble/mwixnet/pull/1

**Comment by boorishpea** [comment]
Please also discuss
https://forum.grin.mw/t/bug-rust-client-crashes-upon-inputting-half-a-slate-pack/9403

**Comment by davidtavarez** [comment]
> Transaction logs need to be generated for public to see. @davidtavarez

https://etherscan.io/tx/0xafebbcf478720ee950f6daf1c839be882b13da464ea187992afd540bc7a28a0f
